### DISCUSSION TITLE
After Update 3.4 --> 4.0.X  unknown service

### DISCUSSION THREAD

**Discussion by kroessig** [Blackymas/NSPanel_HA_Blueprint#1153] [Q&A]
Hello,
i found two notifications about unknown services (and i think they showed the first time after the last update to version 4.0.X).

First:
German text:
Die Automatisierung "NSPanel Configuration" (automation.nspanel_configuration) hat eine Aktion, die einen unbekannten Dienst aufruft: esphome.panelwozi_set_component_color.
Translated with google:
The NSPanel Configuration automation (automation.nspanel_configuration) has an action that calls an unknown service: esphome.panelwozi_set_component_color.

Secound:
Die Automatisierung "NSPanel Configuration" (automation.nspanel_configuration) hat eine Aktion, die einen unbekannten Dienst aufruft: esphome.panelwozi_page_home.
Translated with google:
The NSPanel Configuration automation (automation.nspanel_configuration) has an action that calls an unknown service: esphome.panelwozi_page_home.

I once read something that says you should set a "main or start"-page or something like that after the update, but I can't find the information anymore. Maybe she has nothing to do with it.

Can i ignore these notifications?

**Comment by edwardtfn** [comment]
> Can i ignore these notifications?

No. 😂


You have a version mismatch. Most likely an outdated blueprint
If you restart your panel, in the blue boot screen you will see the versions for TFT, ESPHome and Blueprint. The latest for all is 4.0.2.

https://github.com/Blackymas/NSPanel_HA_Blueprint/wiki/(EN)-HowTo---All-important-thing-you-should-know#2-update-blueprint

**Comment by kroessig** [comment]
Hello,

i used the shown instructions above, and all versions are 4.0.1.
![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/139384422/b03965a6-256f-49f9-8a23-8970b3e0dbf3)

I found no bug in this version - so i do not update to 4.0.2 yet. I clicked that I have fixed the error and let's see what else happens

**Reply by edwardtfn** [reply]
4.02 fixes a critical bug for US version, which is not your case.
For EU version or US landscape would be only one issue with the media player, but that would be gone if you update just the Blueprint side. It's fine if you use blueprint v4.0.2 with TFT and ESPHome on v4.0.1.

## Overview of all changes

| Change | Criticality | Page/area | Components affected |
| :-- | :--: | :--: | :--: |
| Fix incorrect addressing on short press of button02 at buttonpage03 | Critical | `buttonpage03` | TFT (US model only) |
| Fix incorrect service call for Media Player on short press at button pages | Medium | `buttonpageXX` | Blueprint |
| Update to customization instructions | Low | N/A | GitHub page (docs) |
| Use UTC for date selection on weather pages | Low | `weatherXX` | Blueprint |
| Removed unnecessary refresh of relay's chips | Low | `home` | ESPHome |
